### PR TITLE
add Modal Page

### PR DESCRIPTION
**Pull request by hsnice16**
added Modal Component.

Live: https://sad-pare-58cb2c.netlify.app/pages/components/modal.html

Please review this PR, and give your feedback.

### CODE REVIEW

**Review by ApurvaSawant11 — APPROVED** [review]
Looks good to merge!

File: documentation.css [hunk 1]
```diff
@@ -132,6 +132,14 @@ footer {
   width: 30rem;
 }
 
+#modal-component .modal {
+  z-index: 0;
```
Comment (Hamza-777) [inline documentation.css:136]: if z-index is 0 then no need to specifically provide one.

**Review by Hamza-777 — APPROVED** [review]
Looks good.

File: documentation.css (hunk 1 above)
Comment (hsnice16) [reply documentation.css:136]: Thanks for checking this, Hamza. Actually, I am overriding the z-index property here, default it has a z-index set and I am removing that.

**Comment by hsnice16** [comment]
Thank you for reviewing.

File: documentation.css (hunk 1 above)
Comment (Hamza-777) [reply documentation.css:136]: Okay good work himanshu bhai

**Pull request merged**